tests/test_parser.py

```diff
@@ -19,20 +19,27 @@
 
 
 class TestParse(unittest.TestCase):
+    def assertEqualMetrics(self, first, second, msg=None):
+        super(TestParse, self).assertEqual(first, second, msg)
+
+        assert type(first) == type(second)
+        for a, b in zip(first, second):
+            for sa, sb in zip(a.samples, b.samples):
+                assert sa.name == sb.name
```

Comment:
Can you add a comment as to why you're doing this?

Reply:
That's a bit lengthy, you only need to say that you're checking the type is correct.

Reply:
done